### PR TITLE
WIP: Initial schedule release

### PR DESCRIPTION
**Pull request by hackygolucky**
Attempting to get the initial schedule out. Please take a look and a hack! This needs to go up ASAP and we'll share the speaker schedule with the speakers before releasing that granular of a schedule.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -294,11 +294,41 @@ <h3 id="tickets_cta"><a href="https://ti.to/event-loop/cascadiajs-2014">Buy your
...
+                                    <th class="schedule_time">7:00pm</th>
+                                    <td class="schedule_event">Opening party and JS Art Installation</td>
+                                </tr>
+
+                            </tbody>
+                        </table>
+
+                    </div>
                     <table>
+                        <h3>Talk Days</h3>
```
Comment (chrisdickinson) [inline index.html:331]: this is not a valid place to put an h3, afaik.

**Comment by chrisdickinson** [comment]
I will take a look at this later today.

File: index.html (hunk 1 above)
Comment (hackygolucky) [inline index.html:331]: For curiosity's sake, what do you mean by 'not valid'? Will it not render correctly in some browsers? It -just worked- when I checked on it locally.

Comment (junosuarez) [inline index.html:331]: just switch the order of this tag and the table tag, so it goes `<h3>...</h3><table>...` and you're good to go

**Pull request closed without merging**